erpnext/stock/dashboard/item_dashboard_list.html

```diff
@@ -40,18 +40,20 @@
 				</span>
 			</div>
 			<div class="col-sm-2 text-right" style="margin-top: 8px;">
```

Comment:
Add if condition ``` {% if can_write %}``` before div

Reply:
If can_write is false, col-sm-2 will not render and it may disrupt the 12 column layout. Thoughts?